bin/ch-pack-image

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
+
+libexec=$(cd "$(dirname "$0")" && pwd)/../libexec/charliecloud
+. "${libexec}/base.sh"
+
+# shellcheck disable=SC2034
+usage=$(cat <<EOF
+Flatten a builder image into a Charliecloud packed file.
+
+Usage:
+
+  $ $(basename "$0") [-b BUILDER] [--fmt CH_PACK_FMT] IMAGE OUTDIR [ARGS ...]
+
+You must have sufficient privilege (via sudo) to run the Docker commands.
+EOF
+)
+
+parse_basic_args "$@"
+
+# Kludgey argument parsing.
+while true; do
+    case $1 in
+        -b|--builder)
+            shift
+            CH_BUILDER=$1
+            shift
+            ;;
+        -b=*|--builder=*)
+            CH_BUILDER=${1#*=}
+            shift
+            ;;
+        --pack-fmt)
```

Comment:
This argument, `--pack-fmt`, is inconsistent with what usage specifies (`--fmt`).

FWIIW I think `--format` is better.

Reply:
`--pack-fmt` makes it consistent with `ch-test`

Reply:
> `--pack-fmt` makes it consistent with `ch-test`

I am pointing out a discrepancy in the usage output and your argument handling. Your usage message tells users to pass `--fmt` but your argument handling is looking for `--pack-fmt`.